deploy/cephfs/kubernetes/csi-cephfsplugin.yaml

```diff
@@ -164,6 +166,10 @@ spec:
           emptyDir: {
             medium: "Memory"
           }
+        - name: ceph-csi-mountinfo
+          hostPath:
+            path: /var/lib/kubelet/plugins/cephfs.csi.ceph.com/mountinfo
```

Comment:
can we get this path from the `driverName` var?

Reply:
This is just plain YAML document, so I'm not sure that's possible. I did use `driverName` in the Helm chart though.

Reply:
actually it can be picked from ENV var even here.